stdlib/xml/etree/ElementTree.pyi

```diff
@@ -313,9 +313,11 @@ class TreeBuilder:
         def __init__(self, element_factory: _ElementFactory | None = ...) -> None: ...
 
     def close(self) -> Element: ...
-    def data(self, __data: str | bytes) -> None: ...
-    def start(self, __tag: str | bytes, __attrs: dict[str | bytes, str | bytes]) -> Element: ...
-    def end(self, __tag: str | bytes) -> Element: ...
+    def data(self, __data: str) -> None: ...
+    # tag and attrs are passed to the element_factory, so they could be anything
+    # depending on what the particular factory supports.
+    def start(self, __tag: Any, __attrs: dict[Any, Any]) -> Element: ...
```

Comment:
might be worth keeping the comment, but only changing the type if someone actually complains?

Reply:
The existing type is pretty hard to achieve due to invariance, so I'd rather not keep it.